### PR TITLE
New adapter - MgidX

### PR DESCRIPTION
**Pull request by xmgiddev**

### CODE REVIEW

**Comment by gargcreation1992** [comment]
@xmgiddev PR checks are failing. Could you please check once?

**Comment by xmgiddev** [comment]
@gargcreation1992 fixed

**Review by gargcreation1992 — COMMENTED** [review]
Just one comment around setting bid type.

File: static/bidder-info/mgidX.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+endpoint: "https://us-east-x.mgid.com/pserver"
+maintainer:
+  email: "prebid@mgid.com"
```
Comment (gargcreation1992) [inline static/bidder-info/mgidX.yaml:3]: Sent email to verify this email address. Please reply with `Received`.

File: adapters/mgidX/mgidX.go [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+	if index, found := impMap[impID]; found {
+		if index.Banner != nil {
+			return openrtb_ext.BidTypeBanner, nil
+		}
+		if index.Video != nil {
+			return openrtb_ext.BidTypeVideo, nil
+		}
+		if index.Native != nil {
+			return openrtb_ext.BidTypeNative, nil
+		}
```
[the remark this answers is not in the document]
Comment (xmgiddev) [reply adapters/mgidX/mgidX.go]: changed

File: static/bidder-info/mgidX.yaml (hunk 1 above)
Comment (gargcreation1992) [reply static/bidder-info/mgidX.yaml:3]: email received.

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**